pkg/operator/encryption/controllers/migration_controller.go

```diff
@@ -223,12 +223,16 @@ func (c *migrationController) migrateKeysIfNeededAndRevisionStable() (migratingR
 		}
 
 		// idem-potent migration start
+		c.eventRecorder.Eventf("EncryptionMigrationStarted", fmt.Sprintf("Started migration for %s", gr.String()))
```

Comment:
this changed. This does not start migration. Ensure is idempotent. Would move the event into the migrator.